api/v1/accounts.js

```diff
@@ -72,6 +72,10 @@ router.post('/register', (req, res) => {
    * http://play.edenxi.com/tools
    */
   try {
+    const servercodequery = 'SELECT code FROM code;';
+    const servercode = await req.app.locals.query(servercodequery);
+    const codebool = [servercode === code];
```

Comment:
Why is this a boolean array?

Reply:
I wasn't certain if the structure was causing conflicts. It ended up like this but works as three constants for its three separate functions.